content/log/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+{{< img src="log/index/pipeline_sketch.png" alt="Pipelines sketch" >}}
+
+## Getting started with the Agent
+
+Collecting logs is **disabled** by default in the Datadog-Agent.
+In order to start to gather your logs you need to:
+
+1. [Install the latest Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+/////\\\\\\<br>
+/////\\\\\\<br>
+/////\\\\\\Add specific steps once we have the definitive process<br>
+/////\\\\\\<br>
+/////\\\\\\<br>
+
+## Enabling log collection from integrations
+
+To start collecting logs for an integration, you need to uncomment the log sections in your integration yaml file, and then set its parameters values according to your infrastructure.
+
+If you don't find a log section in your file you need to use the custom file configuration below instead. 
+
+## Custom log collection
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit those parameter:
+* `type` : (manatory) type of log input source (**tcp** / **udp** / **file**)
+* `port` / `path` : (mandatory) option depending on `type`.
+* `service` : (mandatory) name of the service owning the log
+* `source` : (mandatory) attribute that defines which integration is sending the logs. It can be a custom one if the integration does not exist
+* `sourcecategory` : (optional) Multiple value attribute. Can be used to refine the source attribtue. Example: source:mongodb, sourcecategory:db_slow_logs
+* `tags`: (optional) add tags to each logs collected.
+
```

Comment:
Suggestion. 
During the beta and for custom yaml files, we need to have at least one "instance" defined which is why we have the "whatever:anything". 
However this parameter is not necessary when using an already existing yaml file with instances defined.
Not sure how to explain that well. Suggestion welcome.

"We will now see configuration examples. 
Please note that for the yaml file to be considered as correct by the agent, you need to add the “init_config” section and have at least one “instance" defined as we show below."

Reply:
👍